src/objects/deck/deck.scss

```diff
@@ -60,3 +60,30 @@
 .o-deck--align-start {
   align-items: start;
 }
+
+/**
+ * Block alignment adjustments
+ *
+ * 1. Set inline padding to match `o-container__fill-pad`.
+ *    We're using the container tokens here, but ideally we'd make a
+ *    token for the whole `fluid-clamp()` formula, but our token system
+ *    doesn't support using Sass formulas that way. This should be okay,
+ *    since we're still matching the container's tokens.
```

Comment:
You could create a mixin instead if you wanted to keep things DRY.

Reply:
Good idea, I'll do that.